var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -61,14 +62,11 @@ def headers(self):
         hlist.directories = [incdir]
         return hlist
 
-    build_directory = 'spack-build'
-
```

Comment:
Am I getting this wrong or are we now building in-source? Why was this necessary?

Reply:
Yeah, sorry, this was unintended